calibration/CTDMOG/CGINS-CTDMOG-13645__20170412.csv

```diff
@@ -1,25 +1,25 @@
 serial,name,value,notes
-37-13645,CC_a0,-1.007343e-004,
+37-13645,CC_a0,-1.007343e-004,source_file 3305-00101-00527-A_SN_37-13645_QCT_Results_CTDMO-G.log
 37-13645,CC_a1,3.065844e-004,
 37-13645,CC_a2,-4.485299e-006,
 37-13645,CC_a3,2.020581e-007,
 37-13645,CC_cpcor,-9.57E-08,
 37-13645,CC_ctcor,3.25E-06,
-37-13645,CC_g,-9.659591e-001,
+37-13645,CC_g,-9.659592e-01,
 37-13645,CC_h,1.390512e-001,
 37-13645,CC_i,-1.192362e-004,
 37-13645,CC_j,2.873961e-005,
 37-13645,CC_p_range,1450,
-37-13645,CC_pa0,3.378686e-002,
-37-13645,CC_pa1,4.595135e-003,
-37-13645,CC_pa2,-1.637389e-011,
-37-13645,CC_ptca0,5.240974e+005,
+37-13645,CC_pa0,5.640599e-02,
+37-13645,CC_pa1,4.589910e-03,
+37-13645,CC_pa2,-1.639871e-11,
+37-13645,CC_ptca0,5.241023e+05,
 37-13645,CC_ptca1,-9.905576e-001,
 37-13645,CC_ptca2,7.534114e-002,
-37-13645,CC_ptcb0,2.502362e+001,
-37-13645,CC_ptcb1,1.325000e-003,
+37-13645,CC_ptcb0,2.526488e+01,
+37-13645,CC_ptcb1,-2.500000e-05,
 37-13645,CC_ptcb2,0.00E+00,
 37-13645,CC_ptempa0,-6.758949e+001,
 37-13645,CC_ptempa1,5.122162e-002,
-37-13645,CC_ptempa2,-5.596463e-007,
-37-13645,CC_wbotc,3.7924e-007,
+37-13645,CC_ptempa2,-5.596464e-07,
+37-13645,CC_wbotc,3.792352e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT